src/main.cpp

```diff
@@ -1,5 +1,7 @@
 #define LONG_PRESS_MS 1000      // How much ms you should hold button to switch to "long press" mode (tune blinds position)
 #define MAX_STEPPERS_COUNT 4    // This limited by available pins usually
+#define SINGLE_BUTTON_CONTROL   // If you use only one button to move Up/Down
```

Comment:
Don't make this mode as default please. Anyone who already use this codebase shouldn't broke their installations after your changes.
So, who wants "single button control" can enable this option (at platformio env for example), all other - will keep current settings.